src/datomic_rest_api/rest/gene.clj

```diff
@@ -82,7 +82,7 @@
   {:data
    (let [db   (d/entity-db gene)
          cds  (:gene/corresponding-cds gene)
-         data {:defined_by_mutation (not (empty? (:variation.gene/_gene gene)))
+         data {:defined_by_mutation (if (not (empty? (:variation.gene/_gene gene))) 1 0)
```

Comment:
You are correct. The reason why would be because Perl does not have boolean variable types. This is something we can change if we start querying the API from the browser (JavaScript).